iXBRLViewerPlugin/viewer/src/js/search.js

```diff
@@ -13,42 +13,49 @@ export class ReportSearch {
         var dims = {};
         var facts = this._reportSet.facts();
         this.periods = {};
-        for (var i = 0; i < facts.length; i++) {
-            var f = facts[i];
-            var doc = { "id": f.vuid };
-            var l = f.getLabel("std");
-            doc.concept = f.conceptQName().localname;
-            doc.doc = f.getLabel("doc");
-            doc.date = f.periodTo();
-            doc.startDate = f.periodFrom();
-            var dims = f.dimensions();
-            for (var d in dims) {
-                if (f.report.getConcept(d).isTypedDimension()) {
-                    if (dims[d] !== null) {
-                        l += " " + dims[d];
+        // Add hidden facts to index later, so that they appear later in the
+        // default search
```

Comment:
Interesting, I usually think of hidden facts as being logically before all other facts, I guess because I usually think of `<ix:hidden>` as being at the top of the iXBRL file.

Reply:
Likewise, and the old behaviour was effectively just following document order.

But, I can see how the old behaviour was confusing to newcomers to iXBRL, particularly now that search is the default pane, as the visible results don't cause things in the viewer pane to be highlighted.